lops/lop-domain-r5-zynqmp.dts

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (c) 2019,2020 Xilinx Inc. All rights reserved.
+ *
+ * Author:
+ *       Bruce Ashfield <bruce.ashfield@xilinx.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
```

Comment:
There's a lot of extra stuff in this lops file, since it was driving a lot of testing early on. I'll merge this as-is, but then we can go through and drop all the extra bits in future commits.